integration/testnet/network.go

```diff
@@ -699,13 +657,26 @@ func BootstrapNetwork(networkConf NetworkConfig, bootstrapDir string) (*flow.Blo
 		return nil, nil, nil, nil, err
 	}
 
+	numViewsInDKGPhase := uint64(100)
+	numViewsInStakingAuction := uint64(5)
+	numViewsInEpoch := uint64(350)
+
 	// generate epoch service events
 	epochSetup := &flow.EpochSetup{
-		Counter:      epochCounter,
-		FinalView:    root.Header.View + leader.EstimatedSixMonthOfViews,
-		Participants: participants,
-		Assignments:  clusterAssignments,
-		RandomSource: randomSource,
+		Counter:            epochCounter,
+		FirstView:          root.Header.View,
+		DKGPhase1FinalView: root.Header.View + numViewsInStakingAuction + numViewsInDKGPhase,
+		DKGPhase2FinalView: root.Header.View + numViewsInStakingAuction + numViewsInDKGPhase*2,
+		DKGPhase3FinalView: root.Header.View + numViewsInStakingAuction + numViewsInDKGPhase*3,
+		FinalView:          root.Header.View + numViewsInEpoch - 1,
```

Comment:
double check these values ("final view" vs "length in views" between protocol and smart contract configs)